src/data/uma-nickname.csv

```diff
@@ -118,7 +118,7 @@
 "意外性","Unpredictable"
 "世代のキング","King of the Era"
 "名優","Great Actor"
-"重賞ウマ娘","Big Prize Girl"
+"重賞ウマ娘","Graded Race Girl"
```

Comment:
Don't like this one, the former one sounds much better

Reply:
Can revert this, no problem.